L1Trigger/TrackFindingTracklet/interface/Settings.h

```diff
@@ -277,6 +277,14 @@ namespace trklet {
     void setStripLength_PS(double stripLength_PS) { stripLength_PS_ = stripLength_PS; }
     void setStripLength_2S(double stripLength_2S) { stripLength_2S_ = stripLength_2S; }
 
+    int numrinvbins() const { return numrinvbins_; }
+    void setNumrinvbins(int numrinvbins) { numrinvbins_ = numrinvbins; }
```

Comment:
I think function setNumrinvBins() is never called. So can we delete it?

Reply:
These have been deleted.